packages/blade/src/components/Box/__tests__/Box.ssr.test.tsx

```diff
@@ -61,9 +73,17 @@ describe('<Box />', () => {
       );
     } catch (err: unknown) {
       expect(err).toMatchInlineSnapshot(`
-        [Error: [Blade: Box]: Oops! Currently you can only use \`transparent\`, \`surface.background.*\`, and \`brand.*\` tokens with backgroundColor property but we received \`red\` instead.
+        [Error: The above error occurred in the <Box> component:
 
-         Do you have a usecase of using other values? Create an issue on https://github.com/razorpay/blade repo to let us know and we can discuss ✨]
+            at backgroundColor (/Users/anurag.hazra/Documents/razorpay/projects/blade/packages/blade/src/components/Box/Box.tsx:222:13)
+            at children (/Users/anurag.hazra/Documents/razorpay/projects/blade/packages/blade/src/components/BottomSheet/BottomSheetStack.tsx:26:3)
```

Comment:
This path is specific to your machine. need to normalize to check for substring only.

Reply:
Oops! checking. 

This should even error like this.

Reply:
Thanks for catching this, removed this tests, we should not even need to test for these since this exact test are already present on the .web.ts files. 

SSR test should contain SSR related flows & smoke tests.